allennlp/service/predictors/wikitables_parser.py

```diff
@@ -36,3 +43,48 @@ def _json_to_instance(self, json_dict: JsonDict) -> Tuple[Instance, JsonDict]:
                                                          tokenized_question=tokenized_question)
         extra_info = {'question_tokens': tokenized_question}
         return instance, extra_info
+
+    @overrides
+    def predict_json(self, inputs: JsonDict, cuda_device: int = -1) -> JsonDict:
+        instance, return_dict = self._json_to_instance(inputs)
+        outputs = self._model.forward_on_instance(instance, cuda_device)
+
+        os.makedirs('data', exist_ok=True)
```

Comment:
I'd probably put this code in the constructor instead of here.  But, maybe it's not that big of a deal, because you're only calling `wget` if the file isn't there...  Still, seems nicer to take this hit at startup instead of during serving.

Reply:
done